ReleaseProcessAutomation/Steps/PipelineSteps/ReleaseAlphaBetaStep.cs

```diff
@@ -94,7 +95,34 @@ public void Execute (SemanticVersion nextVersion, string? commitHash, bool pause
 
     var nextJiraVersion = InputReader.ReadVersionChoice("Please choose next version (open JIRA issues get moved there):", nextPossibleJiraVersions);
 
-    _ijIraFunctionality.CreateAndReleaseJiraVersion(nextVersion, nextJiraVersion);
+    _jiraFunctionality.CreateAndReleaseJiraVersion(nextVersion, nextJiraVersion);
+
+    var currentFullVersion = nextVersion.GetCurrentFullVersion();
+    try
+    {
+      var jiraClosedIssuesWithFixVersion = _jiraFunctionality.GetClosedVersionsSummaryWithExactFixVersion(currentFullVersion);
+      
+      if (jiraClosedIssuesWithFixVersion.Any())
+      {
+        foreach (var issue in jiraClosedIssuesWithFixVersion)
```

Comment:
We are enumerating over the same enumerable twice, thus calculating the same thing twice. Maybe we should convert the enumerable to an array to prevent this.